src/__tests__/components/Row.test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Row from '../../components/row';
+import {shallow} from 'enzyme';
+
+const state = {
+  active: '1',
+  confirmed: '11',
+  deaths: '0',
+  deltaconfirmed: '0',
+  deltadeaths: '0',
+  deltarecovered: '0',
+  lastupdatedtime: '30/03/2020 11:27:27',
+  recovered: '10',
+  state: 'Andaman and Nicobar Islands',
+  statecode: 'AN',
+};
+
+const districts = {
+  'South Andaman': {confirmed: 5, lastupdatedtime: '', delta: {confirmed: 0}},
+  'North and Middle Andaman': {
+    confirmed: 1,
+    lastupdatedtime: '',
+    delta: {confirmed: 0},
+  },
+  Unknown: {confirmed: 5, lastupdatedtime: '', delta: {confirmed: 0}},
+};
+
+let wrapper;
+
+beforeAll(() => {
+  wrapper = shallow(
+    <Row state={state} districts={districts} index={1} total={false} />
+  );
+});
+
+describe('Row component', () => {
+  test('State/UT details', () => {
+    const stateSelector = wrapper.find('tr.state');
+    const cells = stateSelector.find('td');
+
+    const stateName = cells.at(0).text();
+    const confirmed = cells.at(1).text();
+    const active = cells.at(2).text();
+    const recovered = cells.at(3).text();
+    const deaths = cells.at(4).text();
+
+    expect(stateSelector).toHaveLength(1);
+    expect(cells).toHaveLength(5);
+    expect(stateName).toContain(state.state);
+    expect(confirmed).toEqual('11');
+    expect(active).toEqual('1');
+    expect(recovered).toEqual('10');
+    expect(deaths).toEqual('-');
+  });
+
+  test('State last update data', () => {
+    const stateLastUpdate = wrapper.find('tr.state-last-update');
+
+    expect(stateLastUpdate.text()).toMatch(/14 days ago/i);
```

Comment:
Won't this fail from tomorrow?

Reply:
@shuklaayush the data is hardcoded in the spec:
https://github.com/covid19india/covid19india-react/blob/cd9da8927100ed44b6d6fe6f58e1ff6f2e391501/src/__tests__/components/Row.test.js#L12

Reply:
No, I mean when the difference in dates is calculated inside row.js:
https://github.com/covid19india/covid19india-react/blob/584ec6fecb9cf27d6e79c37dbc0f9967963fbf36/src/components/row.js#L183-L185

Reply:
Oh, I see. Good catch, @shuklaayush 
I may have to stub the date.

Reply:
@shuklaayush done! Mocked the date: https://github.com/covid19india/covid19india-react/pull/1042/commits/f692cda4e2a4c3faf2b493fa6d5a275a1f70330c